_sass/_custom_classes.scss

```diff
@@ -205,3 +209,16 @@ footer {
         border-color: transparent;
     }
 }
+
+#ett-logo {
+    display: none;
+    visibility: hidden;
+}
+
+.nav-break {
+    display: none;
+}
+
+.nav-item:nth-last-child(2) {
+    display: none;
+}
```

Comment:
This is a CSS hack to hide the `GitHub` button in the navbar that otherwise cannot be hidden away.